components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/listeners/GatewayStartupListener.java

```diff
@@ -84,7 +93,14 @@ public void completedServerStartup() {
                 new APIMgtGatewayCacheMessageListener());
         jmsTransportHandlerForEventHub
                 .subscribeForJmsEvents(APIConstants.TopicNames.TOPIC_NOTIFICATION, new GatewayJMSMessageListener());
+    }
 
+    private void deployAPIsInSyncMode() {
```

Comment:
Since this method gets called multiple times, need to put some logs to identify how many times it was called.

Reply:
https://github.com/wso2/carbon-apimgt/pull/8945/commits/1e62c1fcc3a0891eaf379eff1e191d80648f6a14